### DISCUSSION TITLE
Reproduce baseline result

### DISCUSSION THREAD

**Discussion by zoexu119** [snap-stanford/ogb#371] [PCQM4M-LSC]
Hi OGB team! I ran the GIN-virtual baseline using the official code [here](https://github.com/snap-stanford/ogb/tree/master/examples/lsc/pcqm4m-v2) with default hyperparameters. However, the best validation MAE is 0.1159, and it's worse than 0.1083 as you reported. Did you use a different set of hyperparameters? Thank you!

**Comment by weihua916** [comment]
Hi! The default hyper-parameters are what we used. I am not sure what happens, maybe pytorch version? I am not really sure, and I do not have time to investigate it for the time being. Sorry!

**Reply by zoexu119** [reply]
Thank you, Weihua! I believe it's just due to randomness, not a big problem.
